### PR TITLE
Add Cloud DNS ChatOps post

### PR DESCRIPTION
**Pull request by filler**
mtw (my team when) using Carina:

### CODE REVIEW

**Comment by filler** [comment]
Please do hang tight on merging this until I can once and twice over the rendered output from CI.

Also if you have any feedback, come at me.

Thanks, Carina Friends! :sparkling_heart: :sparkles:

**Comment by smashwilson** [comment]
> Please do hang tight on merging this until I can once and twice over the rendered output from CI.

:+1: I've marked your pull request as a work-in-progress. Let us know when you're ready!

I'll be keeping an eye on the PR builds, too - if they get hung up or broken I'll be tracking it down in the logs.

File: _posts/2016-03-15-moving-chatops-into-carina.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+Our initial thought was to bring the Hubot instance and its redis-based datastore into a Rackspace Public Cloud instance.  We started assembling the necessary configuration management code to do the needful.  However at the OpenStack Summit in Tokyo we got to see how easy Carina was to both get started with and to manage the lifecycle of container-based applications.
+
+![A Bright Idea]({% asset_path 2016-03-15-moving-chatops-into-carina/lloyd-idea.gif %})
+
+## Why Not Carina?
+
+Our team rallied behind the idea of moving our ChatOps instance into Carina.  We were reasonably saavy using [Docker](https://www.docker.com/) for things like peer-reviewing the team's [Jenkins Job Builder](http://docs.openstack.org/infra/jenkins-job-builder/) changes.  Also [one of our developers](http://ionrock.org/) was able to bring a [proof of concept of our stack](https://github.com/rackerlabs/designate-carina), powered by [OpenStack Designate](http://docs.openstack.org/developer/designate/), into Carina via [`docker-compose](https://docs.docker.com/compose/).
```
Comment (rgbkrk) [inline _posts/2016-03-15-moving-chatops-into-carina.md]: s/saavy/savvy/

**Comment by smashwilson** [comment]
```
Liquid Exception: the server responded with status 404 in _posts/2016-01-17-deploying-and-building-minecraft-as-a-service.md/#excerpt
```

Grrr. Sorry about that. :wrench:

**Comment by rgbkrk** [comment]
This was a solid read, thank you.

**Comment by filler** [comment]
Pushed updates with feedback + dropped an unused binary blob.  I hope I didnt make Strider angry!  :grimacing:

**Comment by smashwilson** [comment]
Nah, I just deployed the pull request builder yesterday, so we're still ironing out the kinks.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-99becadad9/)

**Comment by smashwilson** [comment]
:metal: Bam, success.

File: _posts/2016-03-15-moving-chatops-into-carina.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
+---
+title: "Moving ChatOps Into Carina"
+date: 2016-03-15 16:00
+comments: true
+author: Nick Silkey <nick.silkey@rackspace.com>
+published: false
```
Comment (smashwilson) [inline _posts/2016-03-15-moving-chatops-into-carina.md]: Ah - you'll need to take the `published` key out, or the PR builder won't publish it, either!

**Comment by filler** [comment]
Do I have to set published to true for it to render in the CI build output?

**Comment by smashwilson** [comment]
Yep! Exactly.

File: _posts/2016-03-15-moving-chatops-into-carina.md (hunk 2 above)
Comment (filler) [inline _posts/2016-03-15-moving-chatops-into-carina.md]: Oh, drop it entirely or just swizzle it to `true`? :stuck_out_tongue_winking_eye:

**Comment by filler** [comment]
Hmm, it doesnt look like CI wants to kick off an addl build with said pushed updates.

**Comment by smashwilson** [comment]
Nah, it's still running - the build for #769 just had to finish first. (It's kind of sluggish...)

**Comment by filler** [comment]
Ahh, ok.  Im used to Jenkins + ghprb where it bumps the 'checks' with 'pending' when you push addl commits which are encapsulated in a PR.

Thanks, @smashwilson!  :muscle:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-30164d2026/). :bow:

**Comment by smashwilson** [comment]
:sparkles:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-a640683c9c/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-7440ade639/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-8c6100784d/)

**Comment by filler** [comment]
Hi Carina Friends!  :sparkles: 

I think this is good to go.  Feel free to give it one more once over, then :shipit: good.

Thanks!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-8c6100784d/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-397c765006/)

**Comment by filler** [comment]
OK SRSLY

`/me &`

**Comment by filler** [comment]
Yay! Thanks all for your help on this. And for making Carina what it is.

**Pull request merged**